src/components/common/ForeignJurisdiction.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-card flat id="foreign-jurisdiction">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3">
+        <label class="title-label">Jurisdiction</label>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <v-select
+          id="country-selector"
+          :items="getCountries()"
+          item-text="name"
+          v-model="selectedCountry"
+          filled
+          return-object
+          placeholder="Jurisdiction Country"
+          :rules="countryRules"
+          menu-props="auto"
+          @input="emitChangedCountry($event)"
+        />
+        <v-select
+          v-if="canadaUsaRegions.length > 0"
+          id="region-selector"
+          :items="canadaUsaRegions"
+          item-text="name"
+          v-model="selectedRegion"
+          filled
+          return-object
+          placeholder="Jurisdiction Region"
+          :rules="regionRules"
+          @input="emitChangedRegion($event)"
+        />
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins } from 'vue-property-decorator'
+import { CountriesProvincesMixin } from '@/mixins'
+
+@Component({})
+export default class ForeignJurisdiction extends Mixins(CountriesProvincesMixin) {
+  selectedCountry = {}
+  selectedRegion = {}
```

Comment:
This component needs the ability to restore foreign jurisdiction from a draft filing. You'll need to accept a couple of props, and you'll need to watch them and update your local variables when they change. (You'll probably also want to use the `immediate: true` option to set your local variables from the props when this component is created.)

Reply:
^^ Also add a unit test(s) for this.